src/test/java/sirius/kernel/xml/XMLReaderTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.xml;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import sirius.kernel.commons.ValueHolder;
+import sirius.kernel.health.Counter;
+import sirius.kernel.SiriusExtension;
+
+import java.io.ByteArrayInputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SiriusExtension.class)
+class XMLReaderTest {
```

Comment:
Dont we want to use Kotlin for the new Tests or is this just a first proof that junit4+junit5 is working?
Because there should already be a branch where some tests are migrated to kotlin somewhere in the repo

Reply:
Again separation of concerns :-) My goal was to switch to Junit5 and staying backward compatible in test. Next step would then either be switching to Gradle or provide Kotlin support in Test. Or even both together.

Reply:
Think both could maybe be done at the same time in a next step by merging this with the changes done in the old gradle/kotlin/junit5 branch